model/src/w3initmd.F90

```diff
@@ -678,28 +706,17 @@ SUBROUTINE W3INIT ( IMOD, IsMulti, FEXT, MDS, MTRACE, ODAT , FLGRD, &
       IFT    = LEN_TRIM(TFILE)
       J      = LEN_TRIM(FNMPRE)
 !
-
-#ifndef W3_CESMCOUPLED
```

Comment:
Please retain this #ifndef W3_CESMCOUPLED.

Reply:
@DeniseWorthen , this correction has been made.